DOC-3108 (#1694)
harness/developer-hub: docs/security-testing-orchestration/sto-techref-category/shared/legacy/_sto-ref-legacy-dataload.md

<details><summary>Dataload scan settings</summary>

The following settings are required for Security steps where the `policy_type` is `dataLoad`.

* `product_site_id` The access ID used to look up a specific product in the scanner. For Burp scans, for example, this is the Burp enterprise site identifier.

>    You might want to create a Harness text secret with your encrypted Id and reference the secret using the format `<+secrets.getValue(project.container-access-id>`. For more information, go to [Add and Reference Text Secrets](/docs/platform/Secrets/add-use-text-secrets).

Reviewer: ```suggestion
   You should create a Harness text secret with your encrypted Id and reference the secret using the format `<+secrets.getValue(project.container-access-id>`. For more information, go to [Add and Reference Text Secrets](/docs/platform/Secrets/add-use-text-secrets).
```
